vault/dendron.topic.schema.cook.first-schema.md

```diff
@@ -0,0 +1,153 @@
+
+We'd like all daily journals to have 5MJ, in this scenario:
+
+- `daily.journal.yyyy.mm.dd`: Logging tasks, and anything else we'd like in our daily journal. Dendron can automatically make these with the [[Daily Journal|dendron://dendron.dendron-site/dendron.topic.special-notes#daily-journal]] command (`Ctrl+Shift+I` / `Cmd+Shift+I`).
+
+## Examples
+
+- [5MJ Template](https://github.com/dendronhq/templates/tree/main/templates/templates.journal.5mj.md)
+
+## Creating the 5MJ template
+
+- Open [[lookup|dendron://dendron.dendron-site/dendron.topic.lookup]] (`Ctrl+L` / `Cmd+L`)
+- Create a note called `templates.journal.daily.5mj`
+  - This creates a `templates.journal.daily` hierarchy if there wasn't one, and a new note called `5mj`
+
+Enter in the following markdown:
+
+```markdown
+<!--
+Based on the journaling method created by Intelligent Change:
+- [Intelligent Change: Our Story](https://www.intelligentchange.com/pages/our-story)
+- [The Five Minute Journal](https://www.intelligentchange.com/products/the-five-minute-journal)
+-->
+
+## Morning
+
+<!-- Fill out this section after waking up -->
+
+### Gratitude
+
+I am grateful for:
+
+1.
+2.
+3.
+
+### What would make today great?
+
+1.
+2.
+3.
+
+### Daily affirmations
+
+I am...
+
+## Evening
+
+<!-- Fill out this section before going to sleep, reflecting on your day -->
+
+### Amazing things that happened today
+
+1.
+2.
+3.
+
+### How could I have made today even better?
+
+I could have made today better by
+```
+
+You now have a new template with the 5MJ outline.
+
+## 5MJ Schema
+
+> Keep in mind: templates can be used with or without schemas automatically applying them. Running the [[Insert Note|dendron.ref.commands#insert-note]] command is a way to insert a template into the currently opened note.
+
+### Creating the 5MJ Schema
+
+- Open [[schema lookup|dendron://dendron.dendron-site/dendron.topic.lookup.schemas]] (`Ctrl+Shift+L` / `Cmd+Shift+L`)
+- Create a schema called `5mj`
+
+This creates `5mj.schema.yml`, which should have default content looking like the following.
+
+```yml
+version: 1
+imports: []
+schemas:
+  - id: 5mj
+    children: []
+    title: 5mj
+    parent: root
+
+```
+
+#### 5MJ Schema Contents
+
+Replace the content of `5mj.schema.yml` with the following content. Here, we'll make use of [[Inline Schema|dendron://dendron.dendron-site/dendron.topic.schema#inline-schema-anatomy]].
+
+```yml
+version: 1
+schemas:
+# Daily is the top most schema since its parent is 'root' it must have an identifier
+# this identifier 'daily' will be used when using 'Lookup (schema)' command.
+- id: daily
+  parent: root
+  title: daily
+  desc: ""
+  # Children of the top most schema do not need to contain identifier and just 
+  # require a 'pattern' to be set to match the hierarchy of notes.
+  children:
+    - pattern: journal
+      children:
+        - pattern: "[0-2][0-9][0-9][0-9]"
+          children:
```

Comment:
can we use inline schema? we want to replace all existing examples with inline moving forward

Reply:
Ah, isn't this already using inline schema? It's just a modification of what is used an example in [[Inline Schema|dendron://dendron.dendron-site/dendron.topic.schema#inline-schema-anatomy]]

Reply:
yeah this one is. i think i was commenting on an outdated version.